src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+    private String color;

```

Comment:
You don't need this field here - your super class already has it. Please use a proper super constructor call (described in comment below) and remove this field from all subclasses.

Reply:
ok, done